typings/modules/apollo-client/index.d.ts

```diff
@@ -274,12 +274,13 @@ import { Observable, Observer, Subscription } from '~apollo-client/util/Observab
 export class ObservableQuery extends Observable<GraphQLResult> {
     subscribe(observer: Observer<GraphQLResult>): QuerySubscription;
     result(): Promise<GraphQLResult>;
-}
-export interface QuerySubscription extends Subscription {
```

Comment:
I couldn't figure out how to get `typings` to do the right thing here, so I edited manually to get things to pass. How do we regenerate the types here?